### DISCUSSION TITLE
Issue with tool tip arrow. How is the arrow rendered in the tooltip?

### DISCUSSION THREAD

**Discussion by TristanBrotherton** [themesberg/flowbite-svelte#557] [General]
I have a project where I have just imported the tooltip from flowbite-svelte. Everything is working as expected with the exception of the tooltip arrow which is never rendered. In the DOM I can see:
<img width="817" alt="image" src="https://user-images.githubusercontent.com/78862/216718028-fde5fd41-5ad9-4cd3-8a65-2698e201572c.png">

However the arrow is not show. In the code all i see is: https://github.com/themesberg/flowbite-svelte/blob/1d77a9c4cd8d26fab34dc96a5f91813e2985e004/src/lib/utils/Popper.svelte#L146

However in my implementation it doesn't appear to be getting any styling...

**Comment by shinokada** [comment]
Can you show us your code and create a [REPL](https://svelte.dev/repl/hello-world?version=3.55.1)?

**Reply by TristanBrotherton** [reply]
Thank you for the response. I was implementing this by just importing Tooltip.svelte and its dependancies Frame.svelte and Popper.svelte.
I see now that it expects CSS that was defined outside of those components by Flowbite to render the arrow. So I'd have to extract that CSS manually.
